src/amazonica/core.clj

```diff
@@ -21,6 +20,7 @@
              Regions]
            com.amazonaws.client.builder.AwsClientBuilder
            com.amazonaws.client.builder.AwsClientBuilder$EndpointConfiguration
+           com.amazonaws.services.s3.model.CryptoConfiguration
```

Comment:
core doesn't contain any hard dependencies on any of the actual services, that way folks can only import what they need. see https://github.com/mcohen01/amazonica#for-the-memory-constrained as an example

Reply:
That makes a lot of sense, we always use s3, so this wouldn't be problematic for us, but I understand.

Would need to think more on how to avoid that reflection warning during compilation then.

Perhaps the other changes in this PR are still helpful.

Thanks for your time.

Reply:
Took it out here: https://github.com/mcohen01/amazonica/pull/461/commits/73163c90cf808e590fbd37d0fd5243c045eac608